templates/client-auth-n.cfg.xml.erb

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
 <client-auth xmlns="http://<%= scope.lookupvar('::repose::cfg_namespace_host') %>/repose/client-auth/v1.0">
+<%- if @delegating_enabled -%>
```

Comment:
Yeah, the existing lines are in the wrong place.  I'll remove my param and update this file.